pytests/tests/test_check_skip.py

```diff
@@ -8,56 +8,46 @@
 #   Author: Shreenidhi Shedi <sshedi@vmware.com>
 
 import os
-import tempfile
 import pytest
-import shutil
-import subprocess
 
-isNotPhoton = -1
-try:
-    isNotPhoton = subprocess.check_call("uname -a | grep -i photon > /dev/null", shell = True)
-except Exception as e:
-    isNotPhoton = 1
 
-def run_cmd(utils, cmd, retval):
+def run_cmd(utils, opt, retval):
+    cmd = ['tdnf', '--config', '/etc/tdnf/tdnf.conf', 'check']
+    # DIST will be set in docker environment
```

Comment:
If "DIST" is not set, default to photon.

Reply:
Done